integration-test-common/src/test/java/com/datastrato/gravitino/integration/test/util/AbstractIT.java

```diff
@@ -214,6 +217,20 @@ public static void startIntegrationTest() throws Exception {
       }
 
       GravitinoITUtils.startGravitinoServer();
+      
+      Awaitility.await()
+      .atMost(60, TimeUnit.SECONDS)
+      .pollInterval(1, TimeUnit.SECONDS)
+      .until(
+          () -> {
+            boolean isServerUp = isGravitinoServerUp();
+            if (isServerUp) {
+              LOG.info("Gravitino Server is up and running.");
```

Comment:
you can move the log into the function `isGravitinoServerUp`, and the code here can be simplified as:
```
return isGravitinoServerUp();
```

Reply:
OK!